.github/dependabot.yml

```
@@ -19,6 +19,9 @@ updates:
       - dependency-name: actions/setup-python
       - dependency-name: hashicorp/setup-terraform
       - dependency-name: mxschmitt/action-tmate
+      # # Managed by cisagov/skeleton-python-library
+      # - dependency-name: actions/download-artifact
+      # - dependency-name: actions/upload-artifact
```

I think these lines should be uncommented:
```suggestion
      # Managed by cisagov/skeleton-python-library
      - dependency-name: actions/download-artifact
      - dependency-name: actions/upload-artifact
```

---

Ah, this is one where the repo still has a `first-commits` branch.